docs/getting_started/install_with_ddev.md

```diff
@@ -49,7 +49,7 @@ ddev config --database=postgres:14
 To use MySQL instead, run the following command:
 
 ```bash
-ddev config --database=mysql:8.0
+ddev config --database=mysql:8.4
```

Comment:
While changing this, could you also bump the `--nodejs-version` in line 34 to `22`?